integrationTests/singleShard/block/executingMiniblocksSc_test.go

```diff
@@ -160,292 +158,44 @@ func TestProcessesJoinGameOf100PlayersRewardAndEndgame(t *testing.T) {
 
 	initialVal := big.NewInt(10000000)
 	topUpValue := big.NewInt(500)
+	endGameVal := big.NewInt(100)
```

Comment:
this endgame val does not make sense. In endGame we should not send any value in the transaction. Please remove.

Reply:
Removed.